pkg/store/image/image_test.go

```diff
@@ -34,31 +33,27 @@ func TestImageStore(t *testing.T) {
 			RepoTags:    []string{"tag-1"},
 			RepoDigests: []string{"digest-1"},
 			Size:        10,
-			Config:      &imagespec.ImageConfig{},
 		},
 		{
 			ID:          "sha256:2123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef",
 			ChainID:     "test-chain-id-2abcd",
 			RepoTags:    []string{"tag-2abcd"},
 			RepoDigests: []string{"digest-2abcd"},
 			Size:        20,
-			Config:      &imagespec.ImageConfig{},
```

Comment:
Why removing this? Will add it back in my cleanup PR.

Reply:
because the config is in the image struct...

Reply:
Cleanup?

Reply:
Done.